### PR TITLE
Fixed JUM-117: No Password Attempts on Jumbune deployment (Distributi…

### PR DESCRIPTION
**Pull request by avdheshbansal**
@MishraMayank

### CODE REVIEW

File: deploy/src/main/java/org/jumbune/deploy/DeployUtil.java [hunk 1]
```diff
@@ -670,12 +674,51 @@ private static Session validateUserAuthentication(Console console, Map<String, S
...
+							tempSession = SessionEstablisher
+									.establishConnection(username, namenodeIP,
+											new String(password), null);
+						}
+						if (!tempSession.isConnected()) {
+							passwordRetryAttempts = passwordRetryAttempts + 1;
+							passwdRetryAttempLeft = maxPasswdRetryAttempts
+									- passwordRetryAttempts;
+							if (passwdRetryAttempLeft == 0) {
+								verified = true;
```
Comment (impetus-opensource) [inline deploy/src/main/java/org/jumbune/deploy/DeployUtil.java]: We don't need this boolean set operation if we are exiting VM

Comment (MishraMayank) [inline deploy/src/main/java/org/jumbune/deploy/DeployUtil.java]: We don't need this boolean set operation if we are exiting VM

File: deploy/src/main/java/org/jumbune/deploy/DeployUtil.java [hunk 2]
```diff
@@ -629,9 +629,13 @@ private static boolean containsHadoopLib(String location, Session session)
...
 		String privateKeyPath;
-		Session tempSession;
+		Session tempSession = null;
 		boolean sysexit;
-		int retryAttempts=0;
+		boolean verified = false;
+		int passwordRetryAttempts = 0;
+		int passwdRetryAttempLeft = 0;
+		int maxPasswdRetryAttempts = 3;
+		int retryAttempts = 0;
```
Comment (MishraMayank) [inline deploy/src/main/java/org/jumbune/deploy/DeployUtil.java]: We don't need this variable

File: deploy/src/main/java/org/jumbune/deploy/DeployUtil.java [hunk 3]
```diff
@@ -629,9 +629,13 @@ private static boolean containsHadoopLib(String location, Session session)
 	private static Session validateUserAuthentication(Console console, Map<String, String> argMap) throws IOException {
 		char[] password = null;
 		String privateKeyPath;
-		Session tempSession;
+		Session tempSession = null;
 		boolean sysexit;
-		int retryAttempts=0;
+		boolean verified = false;
+		int passwordRetryAttempts = 0;
+		int passwdRetryAttempLeft = 0;
```
Comment (MishraMayank) [inline deploy/src/main/java/org/jumbune/deploy/DeployUtil.java]: We don't need this too...

Comment (MishraMayank) [inline deploy/src/main/java/org/jumbune/deploy/DeployUtil.java]: We don't need this too.

**Pull request merged**